### ISSUE TITLE
support changing output device

### ISSUE THREAD

**Issue by dvdsk** [smasher164/pw-volume#25]
Setting the default output device via `/etc/pipewire/pipewire.conf` or (temporarily) changing it using `pw-cli` is quite involved. Would you be open to adding both these options in pw-volume?

**Comment by smasher164** [comment]
Yeah I'm considering adding this feature now. I think this will involve the ability to query sinks, and setting one. Probably the most straightforward way is to have it list output devices for `pw-volume device output` and set it via `pw-volume device set-output <output>`.
